.github/workflows/create_release_pr.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Set the package version
         run: |
-          npm version $VERSION
+          npm version ${{ github.event.inputs.version }}
```

Comment:
I don't think this is any different (apart from the env variable is supposedly sanitised)

Maybe a mention of the need to specify `major.minor.patch` in the input description field would be useful?

Reply:
Where is $VERSION set?

Reply:
https://github.com/cylc/cylc-ui/blob/11b8f87d506fdf6fec1150f81d3892db966d8ffa/.github/workflows/create_release_pr.yml#L20-L21

Reply:
Magic variable?

Reply:
(Looking back at that action, not sure it exactly works. But it does set the `$VERSION` and `$BRANCH` env vars, so kind of needed for now)